tests/unit/conftest.py

```diff
@@ -246,3 +246,212 @@ def operation_log_schema(monkeypatch):
         'operation_logs/operation_log-v0.0.1.json'
     )
     return get_schema(monkeypatch, schema_in_bytes)
+
+
+@pytest.fixture()
+def marc21_record():
+    """Marc21 record."""
+    return {
+        'leader': '00000cam a2200000zu 4500',
+        '008': '000000|||||||||xx#|||||||||||||||||||||c',
+    }
+
+
+@pytest.fixture()
+def mef_record_1():
```

Comment:
should be `mef_contribution_xx`. The number should be replaced by something informative such as `person`, `organisation`, `conference`, etc. Same for the next fixtures.